apps/ledger-live-mobile/src/families/stellar/MemoTypeDrawer.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { TouchableOpacity, TouchableOpacityProps } from "react-native";
+
+import type { StellarMemoType } from "@ledgerhq/live-common/families/stellar/types";
+import { Icons, Text } from "@ledgerhq/native-ui";
+import Circle from "~/components/Circle";
+import QueuedDrawer from "~/components/QueuedDrawer";
+
+export const MEMO_TYPES = new Map<MemoType, string>([
+  ["NO_MEMO", "stellar.memoType.NO_MEMO"],
```

Comment:
Why don't use enum for "NO_MEMO" etc? :) 

More easier to use and to be sure we have all options!

Reply:
I also wanted to use an enum at first but then I realised that typescript enum can't really be iterated on (I think they lose their type with `Object.values`) without this the code becomes pretty verbose. Plus Mounir mentioned that it would be an anti pattern. Also IMO inferring the type from `StellarMemoType` (defined in Stellar's `types/bridge.ts`) provides a better type safety.